File: JavaScript/8js 2/person_info.js
/*
|--------------------------------------------------------------------------
| Classy Classes
|--------------------------------------------------------------------------
|
*/

class Person {
   constructor(name, age) {
      this.info = `${name}s age is ${age}`;
   }
}


class Person {
   constructor(name, age) {
      this.name = name;
      this.age = age;
   }
   get info() {
      return `${this.name}s age is ${this.age}`;
   }
}


let john = new Person("john", 34);
console.log(john.info, "johns age is 34");

File: Python/8py 2/person_info.py
"""
|--------------------------------------------------------------------------
| Classy Classes
|--------------------------------------------------------------------------
|
"""

class Person:
   def __init__(self, name, age):
      self.info = f"{name}s age is {age}"


class Person(): __init__ = lambda s,n,a: setattr(s,"info",'{}s age is {}'.format(n,a))


class Person:
   def __init__(self, name, age):
      self.name = name
      self.age = age
   @property
   def info(self):
      return '{}s age is {}'.format(self.name, self.age)


john = Person("john", 34)
print(john.info, "johns age is 34")